### PR TITLE
Make bulk load fully work

### PR DESCRIPTION
**Pull request by soothing-rain**
issue: https://github.com/milvus-io/milvus/issues/15604

/kind enhancement

### CODE REVIEW

File: internal/proto/data_coord.proto [hunk 1]
```diff
@@ -45,6 +48,7 @@ service DataCoord {
   rpc SetSegmentState(SetSegmentStateRequest) returns (SetSegmentStateResponse) {}
   // https://wiki.lfaidata.foundation/display/MIL/MEP+24+--+Support+bulk+load
   rpc Import(ImportTaskRequest) returns (ImportTaskResponse) {}
+  rpc UpdateSegmentStatistics(UpdateSegmentStatisticsRequest) returns (Empty) {}
```
Comment (congqixia) [inline internal/proto/data_coord.proto:51]: Looks strange if an grpc will never returns error. Say `datacoord` is not healthy or segment does not exist, an error is needed to notify either the system or the user.

Comment (soothing-rain) [reply internal/proto/data_coord.proto]: Done.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**